cpu/lm4f120/periph/timer.c

```diff
@@ -37,9 +37,9 @@ typedef struct {
     timer_cb_t cb;          /**< timeout callback */
     void *arg;              /**< argument to the callback */
     unsigned int divisor;   /**< software clock divisor */
-} timer_conf_t;
+} timer_params_t;
```

Comment:
It seems to me that the `timer_conf_t` name is consistent with the other names for peripheral configuration. Why change it ?

Reply:
(also means that the `params` array should be `config` as before)